### PR TITLE
*: Ensure the aggregate CO controller populates the status.Versions field

### PR DESCRIPTION
**Pull request by timflannagan**
Ensure the aggregated platform operator CO resource has a populated status.Versions field. By default, the YAML manifest for this resource has a populated status.Version but the aggregate CO controller logic stomps on this value during reconciliation. Update that reconciliation logic to ensure this value is persisted during events.

Signed-off-by: timflannagan <timflannagan@gmail.com>

### CODE REVIEW

**Comment by timflannagan** [comment]
This is a no-op. Adding the required labels

/label px-approved
/label docs-approved
/label qe-approved

File: internal/clusteroperator/release.go
Comment (tylerslaton) [inline internal/clusteroperator/release.go:13]: I wonder if us not being able to find the `RELEASE_VERSION` env var should cause an error or not like it does in [OLM today](https://github.com/operator-framework/operator-lifecycle-manager/blob/b2086bd2c5eb9b60370869757c51fa7908d2cde0/pkg/controller/operators/openshift/helpers.go#L214). Is there a situation where this value not being present is okay and we can set a default?

**Review by tylerslaton — APPROVED** [review]
Not blocking, but I did have a question before merging.

/hold
/lgtm

File: internal/clusteroperator/release.go
Comment (timflannagan) [reply internal/clusteroperator/release.go:13]: I did this so it's a bit easier to run locally without having to explicitly set the environment variable. This seems sufficient for now, but it's probably worth revisiting this decision over time.

**Pull request merged**
